src/rev_ai/apiclient.py

```diff
@@ -260,13 +280,26 @@ def get_captions(self, id_, filename=None, filepath=None):
             url, headers={'Accept': self.rev_captions_content_type})
         response.raise_for_status()
 
-        if filename:
-            path = os.path.join(filepath or '', filename+'.txt')
-            with open(path, 'w+') as f:
-                f.write(response.text)
-
         return response.text
 
+    def get_captions_as_stream(self, id_):
+        """Get the captions output of a specific job and return it as a plain text stream
+            
+        :param id_: id of job to be requested
+        :returns: requests.models.Response HTTP response which can be used to stream
+            the payload of the response
+        :raises: HTTPError
+        """
+        if not id_:
+            raise ValueError('id_ must be provided')
+
+        url = urljoin(self.base_url, 'jobs/{}/captions'.format(id_))
+        response = self.session.get(
```

Comment:
Should we do like we have done in the node sdk and separate out an api helper?

Reply:
It seems like our python sdk (especially the tests suite) would benefit from the same design pattern.